README.md

```diff
@@ -1,7 +1,152 @@
 # @ngrx/db
+[![Join the chat at https://gitter.im/ngrx/db](https://badges.gitter.im/ngrx/db.svg)](https://gitter.im/ngrx/db?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![CircleCI Status for ngrx/db](https://circleci.com/gh/ngrx/db.svg?style=shield&circle-token=af75fce7603493fca82caac8f0c46cd614407e2c
 )](https://circleci.com/gh/ngrx/db)
+[![npm version](https://badge.fury.io/js/%40ngrx%2Fstore.svg)](https://badge.fury.io/js/%40ngrx%2Fdb)
 
-### RxJS powered IndexedDB for Angular2 apps
+## __**USE ME NOW!**__
```

Comment:
I don't think this is necessary, the fact that "Don't use me yet." was removed is sufficient to indicate it is now usable